docs/src/featureToggle.js

```diff
@@ -2,7 +2,7 @@
 module.exports = {
   nav_products: true,
   enable_website_banner: false,
-  enable_blog_index: process.env.NODE_ENV !== 'production' || process.env.PULL_REQUEST,
+  enable_blog_index: true,
```

Comment:
To move to a different PR? I would argue 

```suggestion
```

Reply:
I think cleanup can be in another PR (batch cleanup for unused feature toggles).